AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -16,6 +16,9 @@ turkishairlines.com,atmos-tokyo.com,radarbox.com,ciceksepeti.com,rabota.by,temp-
 ! notifications - part of the functionality
 ! discord.com##div[class^="container"] > nav[class^="wrapper"] + div[class^="base"] > div[class^="notice-"]
 !
+www.glassdoor.com.hk###AppDownloadPromptContainer
```

Comment:
Where do you need the rule?

Reply:
It appears after you close the banner at the top.
![image](https://user-images.githubusercontent.com/129025690/229832860-bae65423-4f78-4393-b4ad-0cd3a6aa546b.png)

But since we don't have a banner at the top now, this popup won't be triggered. This rule becomes useless.